api/services/auth.js

```diff
@@ -1,7 +1,16 @@
-const { hashHelpers, jwtHelpers } = require('../helpers');
+const nodemailer = require('nodemailer');
+const { hashHelpers, jwtHelpers, emailHelpers } = require('../helpers');
+const { ErrorHandler } = require('../middlewares/errorHandler');
+const {
+  ResetPasswordApproveError,
+  ResetPasswordError,
+} = require('./errorHandlers/index');
+
 const errors = require('./errorHandlers/index');
 const db = require('../../database');
 
+const HOST = process.env.CL_HOST || 'http://localhost:3000';
```

Comment:
we are using another env variable, pls check skype to get the newest .env file

Reply:
Done